guild_queue.py

```diff
@@ -0,0 +1,137 @@
+import logging as log
```

Comment:
When bot is added, if `!dc` is executed then it would spam chat with next sounds in queue until is cleared.

Reply:
fixed